package.json

```diff
@@ -28,16 +28,17 @@
     "ot-ui": "^0.0.77",
     "polished": "^2.1.1",
     "query-string": "5",
-    "react": "^16.4.2",
+    "react": "^16.8.6",
     "react-apollo": "^2.1.9",
-    "react-dom": "^16.4.2",
+    "react-dom": "^16.8.6",
     "react-ga": "^2.5.3",
     "react-helmet": "^5.2.0",
     "react-measure": "^2.1.0",
     "react-router-dom": "^4.3.1",
     "react-scripts": "1.1.4",
     "react-select": "^2.0.0",
     "source-map-explorer": "^1.5.0",
+    "typeface-inter": "^3.3.0",
```

Comment:
Since we are using Inter now, shouldn't we remove Roboto?

Reply:
Ok, good idea. It's now done.